### PR TITLE
POM_testPipelineCheckDiscardOld30builds()_testPipelineCheckDiscardOld3builds()

### PR DESCRIPTION
**Pull request by AnastasiiaPV**
AT_055.01 | Pipeline>Check Discard old builds on general configure
https://trello.com/c/U70SkaRQ/945-at05501-pipelinecheck-discard-old-builds-on-general-configure
AT_055.002 | Pipeline>Check Discard old builds on general configure
https://trello.com/c/f6TG4DZE/959-at055002-pipelinecheck-discard-old-builds-on-general-configure

### CODE REVIEW

File: src/test/java/model/ProjectPage.java [hunk 1]
```diff
@@ -61,4 +69,45 @@ public String getBuildNumber() {
...
+    }
+
+    public List<WebElement> getBuildsRowList() {
+        return buildsRowList;
+    }
+
+    public ProjectPage clickMultipleTimesBuildButton(int number) throws InterruptedException {
+        for (int i = 0; i < number; ++i) {
+            clickBuildButton();
+            Thread.sleep(200);
```
Comment (ViktorPodgornov) [inline src/test/java/model/ProjectPage.java:91]: I'm not sure, but in my opinion, Sergey said to use Thread.sleep is bad practice.

Comment (AnastasiiaPV) [reply src/test/java/model/ProjectPage.java:91]: Yes, it is true. But in some cases, only Thread.sleep works. This is such a case. I tried multiple different waits and every time test failed, so I decided to use Thread with only 200millis and this made the test stable and not fall anymore.

Comment (ViktorPodgornov) [reply src/test/java/model/ProjectPage.java:91]: I checked your branch with getWait5() and it worked for me. I put approval but I think Sergey will swear at it))

**Review by ViktorPodgornov — APPROVED** [review]

**Review by EugeniaSrednitskaya — APPROVED** [review]

File: src/test/java/model/ProjectPage.java (hunk 1 above)
Comment (AnastasiiaPV) [reply src/test/java/model/ProjectPage.java:91]: Thank you very much)

**Pull request merged**